lib/auxiliary/id_gen.rb

```diff
@@ -16,9 +16,9 @@ def initialize(range, timeout)
 
     def load_data
       ids = Array.new(@range.last + 1 - @range.first, true)
-      @conn.execute 'CREATE TABLE IF NOT EXISTS ActiveIds('\
-                  'IdNumber INT PRIMARY KEY NOT NULL, PID INT NOT NULL,'\
-                  'Time TIMESTAMP NOT NULL)'
+      @conn.execute 'CREATE TABLE IF NOT EXISTS ActiveIds(' \
```

Comment:
Perhaps it is better to use heredoc instead?

Reply:
Moved to https://github.com/HCK-CI/AutoHCK/pull/161